sms-broadcast/assets/index.html

```diff
@@ -1,108 +1,102 @@
+                    </a>
+                </div>
+                <nav>
+                    <span>Your Twilio application</span>
+                    <aside>
+                        <svg class="icon" role="img" aria-hidden="true" width="100%" height="100%" viewBox="0 0 20 20" aria-labelledby="NewIcon-1577"><path fill="currentColor" fill-rule="evenodd" d="M6.991 7.507c.003-.679 1.021-.675 1.019.004-.012 2.956 1.388 4.41 4.492 4.48.673.016.66 1.021-.013 1.019-2.898-.011-4.327 1.446-4.48 4.506-.033.658-1.01.639-1.018-.02-.03-3.027-1.382-4.49-4.481-4.486-.675 0-.682-1.009-.008-1.019 3.02-.042 4.478-1.452 4.49-4.484zm.505 2.757l-.115.242c-.459.9-1.166 1.558-2.115 1.976l.176.08c.973.465 1.664 1.211 2.083 2.22l.02.05.088-.192c.464-.973 1.173-1.685 2.123-2.124l.039-.018-.118-.05c-.963-.435-1.667-1.117-2.113-2.034l-.068-.15zm10.357-8.12c.174.17.194.434.058.625l-.058.068-1.954 1.905 1.954 1.908a.482.482 0 010 .694.512.512 0 01-.641.056l-.07-.056-1.954-1.908-1.954 1.908a.511.511 0 01-.71 0 .482.482 0 01-.058-.626l.058-.068 1.954-1.908-1.954-1.905a.482.482 0 010-.693.512.512 0 01.64-.057l.07.057 1.954 1.905 1.954-1.905a.511.511 0 01.71 0z"></path></svg>
+                        Live
+                    </aside>
+                </nav>
+            </header>
+        </div>
+        <main>
+            <div class="content">
+                <h1>
+                    <img src="https://twilio-labs.github.io/function-templates/static/v1/success.svg" />
+                    <div>
+                        <p>Welcome!</p>
+                        <p>Your live application with Twilio is ready to use!</p>
+                    </div>
+                </h1>
+                <section>
+                    <h2>Get started with your application</h2>
+                    <p>
+                        Now that your code is deployed, here are the last steps you need to do to finish your app.
+                    </p>
+                    <p>This app allows you to broadcast SMS messages to large numbers of people.</p>
+                    <ol class="steps">
+                        <li>Text anything to your Twilio phone number</li>
+                        <li>
+                            You should receive a response saying "Hello! Text "subscribe" to receive updates,
+                            "stop" to stop getting messages, and "start" to receive them again."
+                        </li>
+                        <li>Reply back with "subscribe" to your Twilio phone number</li>
+                        <li>
+                            You will be subscribed to future broadcasts and receive a response saying "Thanks!
+                            You have been subscribed for updates."
+                        </li>
+                        <li>
+                            From a phone number you have allowed to trigger a broadcast, text
+                            "broadcast &lt;your message here&gt;" to your Twilio phone number
```

Comment:
I still think we can clean up the message here a little, there's no strict requirements on how the message is formatted. Whatever they send, will be broadcast out to their recipients. Maybe something along the lines of 
```
From the phone number that you configured to trigger a broadcast, text any message to your Twilio phone number
```